### DISCUSSION TITLE
Languages status: Hide forks

### DISCUSSION THREAD

**Discussion by iArmanKarimi** [anuraghazra/github-readme-stats#2363] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I was wondering if it's possible to add an argument (like hide=) that would hide forked repos, thus not counting them in the status.

**Comment by rickstaa** [comment, marked as answer]
@iArmanKarimi We currently do not include results from forks in the language card:

https://github.com/anuraghazra/github-readme-stats/blob/9eb0c3cbb40d551b93536c45100994d4c83b8c04/src/fetchers/top-languages-fetcher.js#L25
